Library/ViewModels/SearchViewModel.swift

```diff
@@ -185,10 +186,27 @@ public final class SearchViewModel: SearchViewModelType, SearchViewModelInputs,
 
     self.scrollToProjectRow = self.transitionedToProjectRowAndTotalProperty.signal.skipNil().map(first)
 
-    // KSRAnalytics
+    // Tracking
 
-    viewWillAppearNotAnimated
-      .observeValues { AppEnvironment.current.ksrAnalytics.trackProjectSearchView() }
+    let projectsProperty = Property(
+      initial: [],
+      then: self.projects
+    )
+
+    let discoveryParamsProperty = Property(
```

Comment:
Same idea here. We should remain as close to the style/convention of the project as we can.